app/components/provider_interface/application_card_component.html.erb

```diff
@@ -3,7 +3,9 @@
   <h3 class="govuk-heading-m">
     <%= govuk_link_to candidate_name, provider_interface_application_choice_path(application_choice) %>
     <div class="app-application-card__status">
-      <%= render ProviderInterface::ApplicationStatusTagComponent.new(application_choice: application_choice) %>
+      <div>
```

Comment:
Rather than a spare `<div>`, would it be better to  somehow pass a class down to TagComponent which gives it `display: block`?

Reply:
We have tried that. However, that also stretches it to fill the container, see here: https://github.com/DFE-Digital/apply-for-postgraduate-teacher-training/pull/2035#pullrequestreview-409319761. You do need a separate block wrapping element with the way the HTML is currently structured.